<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="_central_widget">
   <layout class="QGridLayout" name="_grid_layout">
    <item row="0" column="0">
     <widget class="QLabel" name="_label_before">
      <property name="text">
       <string>До обработки:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="_label_input">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="_label_output">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="_label_after">
      <property name="text">
       <string>После обработки:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="_menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>956</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="_menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="_action_open"/>
    <addaction name="_action_save"/>
    <addaction name="_action_save_as"/>
    <addaction name="_action_exit"/>
   </widget>
   <widget class="QMenu" name="_menu_view">
    <property name="title">
     <string>Вид</string>
    </property>
    <addaction name="_action_view_parametres"/>
   </widget>
   <addaction name="_menu"/>
   <addaction name="_menu_view"/>
  </widget>
  <widget class="QToolBar" name="_tool_bar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="_action_open"/>
   <addaction name="_action_save"/>
   <addaction name="_action_search_countours"/>
  </widget>
  <widget class="QStatusBar" name="_status_bar"/>
  <widget class="QDockWidget" name="_dock_widget">
   <property name="windowTitle">
    <string>Параметры обработки</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="_dock_container">
    <layout class="QFormLayout" name="_dock_layout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="5" column="0">
      <widget class="QLabel" name="_label_threshold">
       <property name="text">
        <string>Пороговое значение</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="_label_min_object_size">
       <property name="text">
        <string>Минимальный размер объекта</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="_spin_box_min_object_size">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="_spin_box_threshold">
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::PlusMinus</enum>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="_action_open">
   <property name="icon">
    <iconset resource="res/res.qrc">
     <normaloff>:/open.png</normaloff>:/open.png</iconset>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
   <property name="toolTip">
    <string>Открыть изображение</string>
   </property>
  </action>
  <action name="_action_save">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res/res.qrc">
     <normaloff>:/save.png</normaloff>:/save.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="_action_save_as">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res/res.qrc">
     <normaloff>:/save.png</normaloff>:/save.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить как</string>
   </property>
  </action>
  <action name="_action_exit">
   <property name="icon">
    <iconset resource="res/res.qrc">
     <normaloff>:/exit.png</normaloff>:/exit.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="_action_search_countours">
   <property name="icon">
    <iconset resource="res/res.qrc">
     <normaloff>:/picture.png</normaloff>:/picture.png</iconset>
   </property>
   <property name="text">
    <string>Определить контуры</string>
   </property>
  </action>
  <action name="_action_view_parametres">
   <property name="text">
    <string>Показать параметры настройки</string>
   </property>
   <property name="toolTip">
    <string>Показать параметры обработки</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>_action_exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
